pkg/config/libpodConfig.go

```diff
@@ -0,0 +1,316 @@
+package config
+
```

Comment:
Add a note at the top saying that this is deprecated functionality and should not be used any longer.